docusaurus.config.js

```diff
@@ -11,10 +11,14 @@ const config = {
   favicon: '/assets/website/favicon.ico',
 
   // Set the production url of your site here
-  url: 'https://push.org/',
+  url: process.env.GITHUB_ACTIONS
+    ? `https://push-protocol.github.io`
+    : 'https://push.org/',
   // Set the /<baseUrl>/ pathname under which your site is served
   // For GitHub pages deployment, it is often '/<projectName>/'
-  baseUrl: '/',
+  baseUrl: process.env.GITHUB_ACTIONS
+    ? `/push-website/pr-preview/${process.env.REACT_APP_PREVIEW_BASE}`
```

Comment:
```suggestion
    ? getPreviewBasePath()
```

Reply:
I cannot use previewBasePath like this in the config like, keeps failing. Already tried it. I think it's cause docusaurus js files do not use the type: module constantly. If i add that to package.json, building fails in general